packages/ui/src/GlobalSearch.jsx

```diff
@@ -49,6 +49,8 @@ const useStyles = makeStyles((theme) => ({
     right: "10px",
   },
   modal: {
+    // leave the space that will be hidden behind the logo bar unused
+    paddingTop: (window.configShowTopBar ?? false) ? "50px" : "",
```

Comment:
note: this is bypassing  `src/config.js`, which means we are setting a default value (`false`) in two different places. the alternative is to add an import statement into all pages that make use of `BasePage` :smiling_face_with_tear: